docs/user_guide/Client_Management.rst

```diff
@@ -27,31 +27,15 @@ prior to attempting to install a client.
 Configure DNS
 +++++++++++++
 
-.. only:: simp_4
-
-  In SIMP most managed, system configuration files are pulled over
-  ``rsync`` by Puppet for network efficiency. These managed files include
-  DNS configuration files and can be found at
-  ``/srv/rsync/bind_dns/default``.
-
-.. only:: not simp_4
-
-  In SIMP most managed, system configuration files are pulled over
-  ``rsync`` by Puppet for network efficiency. These managed files include
-  DNS configuration files and can be found at 
-  ``/var/simp/rsync/OSTYPE/MAJORRELEASE/bind_dns/default``.
+In SIMP, most managed, system configuration files are pulled over
+``rsync`` by Puppet for network efficiency. These managed files include
+DNS configuration files and can be found at
+``/var/simp/rsync/OSTYPE/MAJORRELEASE/bind_dns/default``.
```

Comment:
I think the rsync path has environment in it now... @trevor-vaughan do you know?